doc/release_instructions.md

```diff
@@ -104,6 +112,15 @@ OR
 make VERSION_ADDITIONAL=-rc1 release-candidate
  ```
 
+Note: When releasing an alpha/beta/RC version, ensure the helmchart is marked as a pre-release
+
+`etc/helm/pachyderm/Chart.yaml`
+
+```
+annotations:
+  artifacthub.io/prerelease: "true"
+```
```

Comment:
I am guessing this change needs to be pushed to the release branch. I'd add the `git commit -am ""` command and `git push`.

We want release instructions to be complete with each step called out.

Reply:
Sounds good, I'll move it to the section where we edit chart.yaml above and add the git instructions

Reply:
@nitinjainsj  Done